src/sentry/api/serializers/snuba.py

```diff
@@ -262,6 +262,7 @@ def serialize(
         elif "order" in result.data:
             res["order"] = result.data["order"]
         res["isMetricsData"] = result.data.get("isMetricsData", False)
+        res["isMetricsExtractedData"] = result.data.get("isMetricsExtractedData", False)
```

Comment:
This value is added in the next file (run_query from [timeseries_query](https://github.com/getsentry/sentry/blob/d71dab6cab04c54d9e18ccaec60874ee34ad2986/src/sentry/snuba/metrics_performance.py#L189) in `src/sentry/snuba/metrics_performance.py`).